src/app/(registration)/formAppearanceStyles.ts

```diff
@@ -0,0 +1,103 @@
+export const formAppearanceStyles = {
+  elements: {
+    card: {
+      padding: "40px 40px 40px",
+    },
+    logoBox: {
+      height: "47px",
+      justifyContent: "flex-start",
+    },
+    headerTitle: {
+      fontSize: "20px",
+      textAlign: "start",
+      marginBottom: "8px",
+      color: "#222222",
+      fontWeight: "600",
+    },
+    headerSubtitle: {
+      fontSize: "16px",
+      textAlign: "start",
+      color: "#575757",
```

Comment:
thinking about it we might want to use a fn to get these colours by name so if they change hex codes we don't need to update them here